sdc/tests/test_dataframe.py

```diff
@@ -1454,6 +1454,16 @@ def test_impl(df):
 
         pd.testing.assert_series_equal(sdc_func(df), test_impl(df))
 
+    @unittest.skip('DF.getitem unsupported idx as a tuple')
```

Comment:
@Rubtsowa, @densmirn 
I think this test is no longer needed, since it's tested with _test_df_getitem_tuple_idx, am I right?